shell/common/shell_unittests.cc

```diff
@@ -1538,7 +1552,12 @@ TEST_F(ShellTest, Screenshot) {
   sk_sp<SkData> reference_data = SkData::MakeWithoutCopy(
       reference_png->GetMapping(), reference_png->GetSize());
 
-  ASSERT_TRUE(reference_data->equals(screenshot_future.get().data.get()));
+  sk_sp<SkData> screenshot_data = screenshot_future.get().data;
+  if (!reference_data->equals(screenshot_data.get())) {
+    LogSkData(reference_data, "reference");
+    LogSkData(reference_data, "screenshot");
+  }
+  ASSERT_TRUE(reference_data->equals(screenshot_data.get()));
```

Comment:
nit: can we just do something like `ASSERT_TRUE(false)` in the if block above to avoid doing the comparison twice?

Reply:
done

Reply:
We should then drop this line

Reply:
done